README-quick-start.md

```diff
@@ -0,0 +1,605 @@
+    println!("Split a from Axis(0), at index 1:");
+    println!("s1  = \n{}", s1);
+    println!("s2  = \n{}\n", s2);
+}
+```
+The output is:
+```
+a = 
+[[0, 1, 2, 3],
+ [4, 5, 6, 7],
+ [8, 9, 10, 11]]
+
+Split a from Axis(0), at index 1:
+s1  = 
+[[0, 1],
+ [4, 5],
+ [8, 9]]
+s2  = 
+[[2, 3],
+ [6, 7],
+ [10, 11]]
+
+Split a from Axis(0), at index 1:
+s1  = 
+[[0, 1],
+ [4, 1234],
+ [8, 9]]
+s2  = 
+[[2, 3],
+ [6, 7],
+ [10, 11]]
+```
+
+### Deep Copy
+As the usual way in Rust, a `clone()` call will
+make a copy of your array:
+```rust
+use ndarray::prelude::*;
+use ndarray::Array;
+
+fn main() {
+
+    let mut a = Array::range(0., 4., 1.).into_shape([2 ,2]).unwrap();
+    let b = a.clone();
+    
+    println!("a = \n{}\n", a);
+    println!("b clone of a = \n{}\n", a);
+    
+    a.slice_mut(s![1, 1]).fill(1234.);
+    
+    println!("a updated...");
+    println!("a = \n{}\n", a);
+    println!("b clone of a = \n{}\n", b);
+}
+```
+
+The output is:
+```
+a = 
+[[0, 1],
+ [2, 3]]
+
+b clone of a = 
+[[0, 1],
+ [2, 3]]
+
+a updated...
+a = 
+[[0, 1],
+ [2, 1234]]
+
+b clone of a = 
+[[0, 1],
+ [2, 3]]
+```
+
+Noticing that `clone()` will perform a deep copy of the array elements only if you are cloning an owned array (i.e. `Array`).
+Cloning an `ArrayView` does not clone the underlying elements - you are just cloning the view reference (as it happens in Rust when cloning a `&` reference).
+
+## Broadcasting
+
+Arrays support limited broadcasting, where arithmetic operations with array operands of different sizes can be carried out by repeating the elements of the smaller dimension array. 
+
+```rust
+use ndarray::prelude::*;
+
+fn main() {
+    let a = array![
+        [1., 1.], 
+        [1., 2.], 
+        [0., 3.], 
+        [0., 4.]];
+
+    let b = array![[0., 1.]];
+
+    let c = array![
+        [1., 2.], 
+        [1., 3.], 
+        [0., 4.], 
+        [0., 5.]];
+    
+    // We can add because the shapes are compatible even if not equal.
+    // The `b` array is shape 1 × 2 but acts like a 4 × 2 array.
+    assert!(c == a + b);
+}
+```
+
+See [.broadcast()](https://docs.rs/ndarray/latest/ndarray/struct.ArrayBase.html#method.broadcast) for a more detailed description.
+
+And there is a short example of it:
+```rust
+use ndarray::prelude::*;
+
+fn main() {
+    let a = array![
+        [1., 2.],
+        [3., 4.],
+    ];
+    
+    let b =  a.broadcast((3, 2, 2)).unwrap();
+    println!("shape of a is {:?}", a.shape());
+    println!("a is broadcased to 3x2x2 = \n{}", b);
+}
+```
+The output is:
+```
+shape of a is [2, 2]
+a is broadcased to 3x2x2 = 
+[[[1, 2],
+  [3, 4]],
+
+ [[1, 2],
+  [3, 4]],
+
+ [[1, 2],
+  [3, 4]]]
+```
+
+## Want to learn more?
+Please checkout these docs for more information
+* [`ArrayBase` doc page](https://docs.rs/ndarray/latest/ndarray/struct.ArrayBase.html)
+* [`ndarray` for `numpy` user doc page](https://docs.rs/ndarray/0.13.0/ndarray/doc/ndarray_for_numpy_users/index.html)
```

Comment:
both links can link to the latest version - use latest, * or similar for the version?